content/schedule/activities.md

```diff
@@ -7,4 +7,38 @@ duration: 3h30m
 width: 2
 people: all
 ---
-The details fo the Activities will we posted later.
+
+## NWERC activities
+The following excursions are available for the 25th of November. There are limited capacities for each activity, so **full = full**. Registration is possible via the Google Form sent in your e-mail.
+
+The activities will happen in two rounds.
+- In the first round, we are excited to announce two inspiring company talks by our sponsors Huawei and JetBrains.
+- In the second round, you can choose between a set of relaxing activities in Delft.
+
+
+### In the second round, the following options are available (you can choose only one):
+
+---
+
+- Canal Cruise
+
+You will get a boat tour through the Delft canals. Note that the starting time of the activity is the time your boat will leave. There are two rounds; one boat leaves at 16:15 and the other at 17:15. [Google Maps link to where the boat leaves](https://goo.gl/maps/BAkMNtkGcvJneWdR6).
+
+---
+- New Church Visit
+
+You will get free entrance to visit the iconic 'New Church' in the vibrant city center of Delft [(Google Maps link)](https://maps.app.goo.gl/WdHhoEGKJjMLavBf8). This beautiful piece of architecture is the tallest building in Delft.
+
+---
+- Beer Tasting in Local Delft Brewery
+
+You will visit the local Delft brewery 'de Koperen Kat', or, in English, 'the Copper Cat' [(Google Maps link)](https://goo.gl/maps/eXBfGVkCqJzyyMmK9), and you will get to sample a few different beers they make there.
+
+---
+- Other activity
+
+Take matters into your own hands and embark on an adventure in either Delft, Rotterdam or the Hague! You are free to do what you like in the time of the second round.
+We provide an overview of suggestions for other activities [on this map](https://wisv.ch/map). Note that all 'other activities' are self-paid and on your own accord. We do **not** reimburse any travel/entrance cost of this.
```

Comment:
Separate comment: we also do not reimburse travel costs to the Canal/Church/Copper Cat, right? This sentence makes it sound like we do (but of course, we only pay for the entrance fee) :stuck_out_tongue:

Reply:
@mpsijm should be more clear now!

Reply:
Yes, thanks! :smile: